intern/ghost/intern/GHOST_SystemPathsCocoa.mm

```diff
@@ -18,7 +18,7 @@
  * The Original Code is Copyright (C) 2010 Blender Foundation.
  * All rights reserved.
  *
- * 
+ *
```

Comment:
What are these space cleaning ?

Reply:
We try to keep separate code commits from clean-up commits. Additionally, we avoid clean-up commits in code outside of gameengine folder to avoid possible problems with blender-BF merges.
So it is better that you revert those clean-up changes here. 
Other than this, i'm ok with rest of the changes.